pkg/acceptance/testing.go

```diff
@@ -208,3 +210,22 @@ func ExternalProviderWithExactVersion(version string) map[string]resource.Extern
 		},
 	}
 }
+
+// SetV097CompatibleConfigPath sets a new config path in a relevant env variable for a file that is compatible with v0.97.
+func SetV097CompatibleConfigPathEnv(t *testing.T) func() {
```

Comment:
It is a bit misleading that function name says `set` but the truth is that this just provides a function to set it. I would either change the name of the function or - more preferably -  do not return the function but inline the body here.

Reply:
I changed the names to `...Func`. We can't inline them because they need to take `*testing.T` and return `func()` to satisfy `PreConfig`.

Reply:
Well... You can wrap it in func in the PreConfig. This is how we do this in the majority use cases throughout our tests. It's easier then to set multiple things in the PreConfig.

Reply:
Done.